Rakefile

```diff
@@ -5,6 +5,13 @@ require "rake/testtask"
 
 Rake::TestTask.new do |task|
   task.libs << "test"
+  task.test_files = Dir["test/**/*_test.rb"] - Dir["test/**/*_slow_test.rb"]
```

Comment:
The wildcard should consume `_` and alpha numeric characters so it would normally include the slow tests which I don't want for normal CI runs.